flask_discord_interactions/command.py

```diff
@@ -54,8 +54,6 @@ class Command:
         Type for this command (depend on the action in the Discord client).
         The value is in ``ApplicationCommandType``. If omitted, set the default
         value to ``ApplicationCommandType.CHAT_INPUT``.
-    default_permission
```

Comment:
Is it intentional that you only remove the docs and the serialization for `default_permission` but still keep the actual field as argument when defining commands? 
Found occurences in lines 75, 89, 332, 435, 451 and 463 in this file.
Same applies for the `discord.py` file.

Reply:
That was not intentional! Thanks for pointing that out 👍